### PR TITLE
Rename ReadManifestFile and WriteManifestFile to Read/WriteFile

### PR DESCRIPTION
**Pull request by efekarakus**
These methods are generic enough so that we can write and read files that are under the "ecs-project" directory. This PR renames them.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: internal/pkg/cli/pipeline_init.go [hunk 1]
```diff
@@ -193,14 +195,18 @@ func (opts *InitPipelineOpts) createPipelineManifest() (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("marshal manifest: %w", err)
 	}
-	manifestPath, err := opts.manifestWriter.WriteManifest(manifestBytes, pipelineDefaultFilename)
+	manifestPath, err := opts.manifestWriter.WriteFile(manifestBytes, pipelineDefaultFilename)
```
Comment (kohidave) [inline internal/pkg/cli/pipeline_init.go]: Could you add a function, like we have for AppManifestFileName - PipelineFileName()?

Comment (efekarakus) [reply internal/pkg/cli/pipeline_init.go]: Sounds good, I can add it in my follow-up PR that writes the build spec!

File: internal/pkg/cli/pipeline_init.go [hunk 2]
```diff
@@ -42,6 +43,7 @@ type InitPipelineOpts struct {
 	// Interfaces to interact with dependencies.
 	manifestWriter archer.ManifestIO
 	secretsmanager archer.SecretsManager
+	box            packd.Box
```
Comment (sonofachamp) [inline internal/pkg/cli/pipeline_init.go]: Is this being used anywhere?

Comment (efekarakus) [reply internal/pkg/cli/pipeline_init.go]: Gonna use in my upcoming PR! Sorry I shouldn't have included it

**Pull request merged**